mapfishapp/src/main/webapp/WEB-INF/jsp/debug-includes.jsp

```diff
@@ -2,6 +2,7 @@
     <script type="text/javascript" src="<%= context %>/lib/externals/ext/ext-all-debug.js"></script>
     <script type="text/javascript" src="<%= context %>/lib/Ext.ux/lib/Ext.ux.js"></script>
     <script type="text/javascript" src="<%= context %>/lib/proj4js/lib/proj4js-combined.js"></script>
+    <script type="text/javascript" src="<%= context %>/lib/externals/openlayers/lib/Firebug/firebug.js"></script>
```

Comment:
There's already one on line 86 which is IMO better to use, we just have to uncomment it.